lib/lint.ml

```diff
@@ -38,6 +65,18 @@ module Check = struct
     in
     let cmd = ["opam-ci-check"; "lint"; "--opam-repository"; "."] @ changed @ new_ in
     exec ~cwd ~job (cmd |> Array.of_list)
+    >>= function
+    | Error (`Msg err) ->
+      (* The exec function doesn't capture stdout when the command fails; so we
+         parse the command output from the log file, instead. *)
+        let errors = parse_errors_from_log job in
+        (match errors with
+        | Ok errors -> Lwt_result.fail (`Msg errors)
+        | Error (`Msg msg) ->
+          let error_msg = (Fmt.str "%s - %s" err msg) in
```

Comment:
I think we will want to very clearly differentiate this case from errors that report a failing lint to users. Do you think this would help for that purpose?

```suggestion
          let error_msg = (Fmt.str "internal error: %s - %s" err msg) in
```